cdc/model/txn.go

```diff
@@ -6,15 +6,17 @@ import (
 	"github.com/pingcap/tidb/types"
 )
 
-// RawTxn represents a complete collection of Entries that belong to the same transaction
-type RawTxn struct {
-	Ts         uint64
-	IsResolved bool
-	Entries    []*RawKVEntry
+// RawRowGroup represents a complete collection of Entries that belong to the same transaction
+type RawRowGroup struct {
```

Comment:
Note that the word `raw` in `RawTxn` is used to separate it from `Txn`, since it's no longer called a `txn`, you can remove the `raw` part.

Reply:
I will rename Txn to RowGroup in the next PR